DashKit/PodStatusExtension.swift

```diff
@@ -0,0 +1,373 @@
+            return true
+
+        case ( .retrievingPodVersion, .retrievingPodVersion):
+            return true
+
+        case ( .settingPodUid, .settingPodUid):
+            return true
+
+        case ( .programmingLowReservoirAlert, .programmingLowReservoirAlert):
+            return true
+
+        case ( .programmingLumpOfCoal, .programmingLumpOfCoal):
+            return true
+
+        case ( .primingPod, .primingPod):
+            return true
+
+        case ( .checkingPodStatus, .checkingPodStatus):
+            return true
+
+        case ( .programmingPodExpireAlert, .programmingPodExpireAlert):
+            return true
+
+        case ( .podStatus, .podStatus):
+            return true
+
+        case ( .step1Completed, .step1Completed):
+            return true
+
+        default:
+            return false
+        }
+    }
+}
+
+extension ActivationStep2Event : Equatable {
+
+    public var description : String {
+        switch self {
+        case .connecting:
+            return "Pod is connecting"
+
+        case .programmingActiveBasal:
+            return "Programming active basal"
+
+        case .insertingCannula:
+            return "Inserting Cannula."
+
+        case .checkingPodStatus:
+            return "Checking Pod status"
+
+        case .podStatus:
+            return "Pod status"
+
+        case .step2Completed:
+            return "Activation phase 2 completed"
+
+        case .cancelLumpOfCoalProgrammingAutoOff:
+            return "Cancelling lump of coal"
+        }
+    }
+    public static func ==(lhs: ActivationStep2Event, rhs: ActivationStep2Event) -> Bool {
+        switch (lhs, rhs) {
+        case ( .connecting, .connecting):
+            return true
+
+        case ( .programmingActiveBasal, .programmingActiveBasal):
+            return true
+
+        case ( .insertingCannula, .insertingCannula):
+            return true
+
+        case ( .checkingPodStatus, .checkingPodStatus):
+            return true
+
+        case ( .podStatus, .podStatus):
+            return true
+
+        case ( .step2Completed, .step2Completed):
+            return true
+
+        default:
+            return false
+        }
+    }
+}
+
+extension PodState : CustomStringConvertible {
+    public var description : String {
+        switch self {
+        case .uninitialized:
+            return "Pod is uninitialized"
+
+        case .mfgTest:
+            return "MFG Test"
+
+        case .filled:
+            return "Pod is filled"
+
+        case .uidSet:
+            return "UID is set"
+
+        case .engagingClutchDrive:
+            return "Engaging Clutch Drive. \nPlease wait for 35 seconds. Activation is in Progress......"
+
+        case .clutchDriveEnaged:
+            return "Clutch drive is engaged"
+
+        case .basalProgramRunning:
+            return "Basal Program is Running"
+
+        case .priming:
+            return "Pod is priming. \nPlease wait for 10 seconds. Activation is in process......"
```

Comment:
Do any of these `descriptions` for events and states need to be localized?

Reply:
I don't believe any of the activation events or pod state or pod comm state objects are displayed.